packages/cozy-client/src/models/paper.js

```diff
@@ -48,6 +48,7 @@ export const KNOWN_INFORMATION_METADATA_NAMES = [
   'noticePeriod'
 ]
 export const KNOWN_OTHER_METADATA_NAMES = ['contact', 'page', 'qualification']
+export const KNOWN_BILLS_METADATA_NAMES = ['amount', 'subtype', 'employer']
```

Comment:
I don't know if it would be better to call it `KNOWN_BILLS_ATTRIBUTE_NAMES`, rather than sounding like the other “KNOWN_...” names. 🤔